frontend/src/plugins/impl/SwitchPlugin.tsx

```diff
@@ -30,29 +27,20 @@ const SwitchComponent = ({
   data,
 }: IPluginProps<boolean, { label: string | null }>): JSX.Element => {
   const id = useId();
-  const labelElement =
-    data.label === null ? null : (
-      <Label htmlFor={id} className="text-md">
-        {renderHTML({ html: data.label })}
-      </Label>
-    );
 
   return (
-    <div
-      className={cn(
-        labelStyles.labelContainer,
-        "align-middle",
-        "items-start",
-        "gap-x-2"
-      )}
+    <Labeled
+      label={data.label}
+      align="right"
+      id={id}
+      labelClassName="text-md ml-1"
```

Comment:
- let's update the font size to match the other labels, i think that just means removing "text-md"
- let's put the label on the left to match the other plugins?

(These issues may have already existed, but might as well fix them now)

Reply:
i think "controls" (i.e. checkbox and switch) should be on the right side. but we can add an option to expose "align"